api/app/serializers/spree/api/v2/platform/icon_serializer.rb

```diff
@@ -0,0 +1,18 @@
+module Spree
```

Comment:
@MatthewKennedy let's also add serializer specs for this :)

Reply:
Will do that now.